vue-components/src/components/LookupInput.vue

```diff
@@ -23,7 +25,10 @@
 			@select="onSelect"
 			@scroll="onScroll"
 			@esc="onEsc"
+			@keyboard-hover-change="onKeyboardHoverChange"
 			ref="menu"
+			:aria-expanded="showMenu || 'false'"
```

Comment:
Mh, This doesn't work for me here. Could we try to move this `:aria-expanded="showMenu || 'false'"` to the `Input` compnent? At least with orca that seems to work better

Reply:
but what is expanded is the optionsMenu, not the Input. :thinking:

Reply:
I looked into it a bit more and for me it works when added to the `<Input>` if we also add the needed `role="combox"` to it. This is also mentioned in the ticket. And it also works for me here: https://www.w3.org/TR/2019/NOTE-wai-aria-practices-1.1-20190814/examples/combobox/aria1.0pattern/combobox-autocomplete-list.html

Reply:
It works indeed! Is it a combobox though?

Reply:
Yes, I think so. This sounds a lot like what we're building: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/